src/test/java/uk/gov/hmcts/darts/arm/api/impl/ArmDataManagementApiImplTest.java

```diff
@@ -97,4 +103,15 @@ void getResponseBlobData() {
         BinaryData binaryData = armDataManagementApi.getBlobData(blobNameAndPath);
         assertEquals(data, binaryData);
     }
+
+    @Test
+    void downloadArmData() {
+
+        var inputStream = ByteArrayInputStream.nullInputStream();
+        when(armApiService.downloadArmData(EXTERNAL_RECORD_ID, EXTERNAL_FILE_ID)).thenReturn(inputStream);
+
+        InputStream result = armDataManagementApi.downloadArmData(EXTERNAL_RECORD_ID, EXTERNAL_FILE_ID);
+
+        assertEquals(inputStream, result);
```

Comment:
This is doing reference equality. Maybe use apache IO IOUtils.contentEquals(inputStream1, inputStream2);

Reply:
in this case the implementation delegates to `armApiService.downloadArmData` and returns whatever that returns, so I think an equality check is suitable and explains better what's going on?